fmriprep/workflows/confounds.py

```diff
@@ -7,36 +7,46 @@
 Calculates frame displacement, segment regressors, global regressor, dvars, aCompCor, tCompCor
 '''
 from __future__ import print_function, division, absolute_import, unicode_literals
-
 from niworkflows.nipype.interfaces import utility
+from niworkflows.nipype.interfaces import fsl
 from niworkflows.nipype.algorithms import confounds
 from niworkflows.nipype.pipeline import engine as pe
+from niworkflows.nipype.interfaces.fsl import ICA_AROMA as aroma
 from niworkflows.interfaces.masks import ACompCorRPT, TCompCorRPT
+from niworkflows.interfaces import segmentation as nws
 
 from niworkflows.nipype.interfaces.nilearn import SignalExtraction
 from fmriprep.interfaces.utils import prepare_roi_from_probtissue
 
 
-def init_discover_wf(bold_file_size_gb, name="discover_wf"):
+def init_discover_wf(bold_file_size_gb, use_aroma,
+                     ignore_aroma_err, name="discover_wf"):
     ''' All input fields are required.
 
     Calculates global regressor and tCompCor
-        from motion-corrected fMRI ('inputnode.fmri_file').
+    from motion-corrected fMRI ('inputnode.fmri_file').
     Calculates DVARS from the fMRI and an EPI brain mask ('inputnode.epi_mask')
     Calculates frame displacement from MCFLIRT movement parameters ('inputnode.movpar_file')
     Calculates segment regressors and aCompCor
         from the fMRI and a white matter/gray matter/CSF segmentation ('inputnode.t1_seg'), after
         applying the transform to the images. Transforms should be fsl-formatted.
-
+    Optional:
+        Calculates motion related components from ICA_AROMA
```

Comment:
[That is how it is advertised](https://www.ncbi.nlm.nih.gov/pubmed/25770991), but I've heard from colleagues (and from my own data) some of the artifacts removed appear to be more physiological in origin.